lib/src/generator.dart

```diff
@@ -56,6 +56,8 @@ mixin GeneratorContext on DartdocOptionContextBase {
 
   String get templatesDir => optionSet['templatesDir'].valueAt(context);
 
+  String get outputFormat => optionSet['format'].valueAt(context);
```

Comment:
we should keep the name of the variable and the optionSet key the same, for my sanity later.

Reply:
Done, chose `--format`